src/components/ContractCallTransactionsTable.vue

```diff
@@ -2,10 +2,30 @@
   <table>
     <thead>
       <tr>
-        <th>Hash</th>
-        <th>Time</th>
-        <th>Caller</th>
-        <th>Entry Point</th>
+        <th>
+          Hash
+          <hint-tooltip>
+            {{ contractsHints.callTransactionsHash }}
+          </hint-tooltip>
+        </th>
+        <th>
+          Time
+          <hint-tooltip>
+            {{ contractsHints.callTransactionsTime }}
+          </hint-tooltip>
+        </th>
+        <th>
+          Caller
+          <hint-tooltip>
+            {{ contractsHints.callTransactionsCaller }}
+          </hint-tooltip>
+        </th>
+        <th>
+          Entry Point
```

Comment:
```suggestion
          Entrypoint
```

Reply:
Ok, as discussed before I have fixed this in previous PR, but also go aligned in this PR all over the app